CustomCraftSML/SMLHelperItems/CustomFoodCraftable.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace CustomCraft2SML.SMLHelperItems
+{
+    using CustomCraft2SML.PublicAPI;
+    using CustomCraft2SML.Serialization.Entries;
+    using SMLHelper.V2.Assets;
+    using SMLHelper.V2.Crafting;
+    using UnityEngine;
+
+    internal class CustomFoodCraftable : Craftable
+    {
+        public override CraftTree.Type FabricatorType => Path.Scheme;
+        public override string[] StepsToFabricatorTab => Path.Steps;
+        public override TechGroup GroupForPDA => FoodEntry.PdaGroup;
+        public override TechCategory CategoryForPDA => FoodEntry.PdaCategory;
+
+        public override TechType RequiredForUnlock => FoodEntry.UnlockedBy_;
+
+        public override string AssetsFolder => FileLocations.AssetsFolder;
```

Comment:
We will need to also override `IconFileName`. Either the user will provide their own sprite, or we will use one of the sample sprites we will include in the update.